packages/odyssey/src/scss/abstracts/_tokens.scss

```diff
@@ -14,7 +14,7 @@
 
 // Typography
 $body-font-family: 'Whyte', '-apple-system',  'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif;
-$mono-font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace;
+$mono-font-family: 'Inconsolata', 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace;
```

Comment:
Good with this even if we can't ship it at present - progressive enhancement where available.